package.json

```diff
@@ -33,6 +33,7 @@
     "@metalsmith/markdown": "^1.5.0",
     "@metalsmith/permalinks": "^2.4.0",
     "autoprefixer": "^10.4.7",
+    "eslint-config-standard": "^17.0.0",
```

Comment:
I don't know if it matters in this project, but you moved the dependency from devDeps to deps.

Reply:
Whoops, yeah, I'll move it to keep it consistent. Thanks.